impl/sql/mutationstorage/mutation_logs.go

```diff
@@ -79,10 +82,17 @@ func (m *Mutations) Send(ctx context.Context, directoryID string, logID int64, u
 		}
 		updateData = append(updateData, data)
 	}
-	// TODO(gbelvin): Implement retry with backoff for retryable errors if
-	// we get timestamp contention.
-	wm := water.NewMark(uint64(time.Duration(time.Now().UnixNano()) * time.Nanosecond / time.Microsecond))
-	if err := m.send(ctx, wm, directoryID, logID, updateData...); err != nil {
+
+	b := backoff.Backoff{Min: time.Millisecond, Max: time.Second, Factor: 1.2, Jitter: true}
```

Comment:
These parameters seem a bit intense. Maybe start from something like 10 or 100 ms?

Also, I am a bit worried that it can loop forever. But, I guess, some higher level timeout would stop it, right?

Reply:
yes, a higher level timeout will stop it.